src/plugins/multifile/multifile.c

```diff
@@ -102,6 +103,9 @@ static inline Codes rvToRc (int rc)
 	case 1:
 		return SUCCESS;
 		break;
+	case 2:
+		return CACHE_HIT;
+		break;
```

Comment:
Isn't this whole function just a `return rc`? (Assuming rc is valid. The other cases should be asserted anyway.)

Reply:
It seems so ...

Reply:
Ok.